RecoTracker/IterativeTracking/python/iterativeTk_cff.py

```diff
@@ -43,6 +43,25 @@
     InitialStepPreSplitting +
     InitialStep +
     HighPtTripletStep +
+    DetachedQuadStep +
```

Comment:
Please refer to this [talk](https://indico.cern.ch/event/281373/contribution/3/attachments/516584/712714/IterationSpeed4Nov13.pdf) to see why we swapped iterations.